app/models/facility.rb

```diff
@@ -49,6 +49,11 @@ def accounts
   scope :active, :conditions => { :is_active => true }
   scope :sorted, order: :name
 
+  def self.cross_facility
```

Comment:
Should we standardize on `all_facility` vs `cross_facility`? I don't have a preference, but it seems like we should be using the same thing everywhere.